packages/sdks/src/blocks/embed/component-info.ts

```diff
@@ -20,35 +20,6 @@ export const componentInfo: ComponentInfo = {
       required: true,
       defaultValue: '',
       helperText: 'e.g. enter a youtube url, google map, etc',
-      onChange: (options: Map<string, any>): void | Promise<void> => {
-        const url = options.get('url');
-        if (url) {
-          options.set('content', 'Loading...');
-
-          const apiKey = _iframelyApiKey;
```

Comment:
I think you can also delete the `declare const _iframelyApiKey: string;` in these files as a result, no?

Reply:
tough crowd